src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java

```diff
@@ -1623,6 +1623,43 @@ public void makeHttpRequestsShouldNotCreateUserExtTpIdWithAdServerEidSourceIfExt
                         .build());
     }
 
+    @Test
+    public void makeHttpRequestsShouldCreateUserExtEidsithAdServerEidSource() {
```

Comment:
Typo in test name: `ExtEidsith` should be `ExtEidsWith`

Reply:
Nice catch!